### PR TITLE
URI encode ad parameters

### PR DESCRIPTION
**Pull request by MelodyLi2017**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
- removing call to nonexistent 'getUserSync' function
- uriencode ad request params that have curly braces or brackets
- search for ccpa string with the correct key

- adapter maintainer's email: opensource@yieldmo.com

### CODE REVIEW

File: modules/yieldmoBidAdapter.js [hunk 1]
```diff
@@ -39,18 +38,23 @@ export const spec = {
...
+              ? bidderRequest.gdprConsent.gdprApplies
+              : '',
+          cmp:
+            bidderRequest && bidderRequest.gdprConsent
+              ? bidderRequest.gdprConsent.consentString
+              : '',
+        })
+      ),
+      us_privacy:
+        bidderRequest && bidderRequest.uspConsent
```
Comment (HolzAndrew) [inline modules/yieldmoBidAdapter.js:55]: was there a change to the privacy framework (us_privacy vs uspConsent)?

**Comment by HolzAndrew** [comment]
LGTM pending privacy framework question

File: modules/yieldmoBidAdapter.js (hunk 1 above)
Comment (MelodyLi2017) [reply modules/yieldmoBidAdapter.js:55]: http://prebid.org/dev-docs/modules/consentManagementUsp.html#bidder-adapter-us-privacy-integration says bidderRequest.uspConsent

Comment (MelodyLi2017) [reply modules/yieldmoBidAdapter.js:55]: I'll also make a test page since we don't have one of those :U

Comment (HolzAndrew) [reply modules/yieldmoBidAdapter.js:55]: ah i see. sounds good

File: modules/yieldmoBidAdapter.js [hunk 2]
```diff
@@ -99,17 +105,8 @@ export const spec = {
...
-      return [
-        {
-          type: 'iframe',
-          url: SYNC_ENDPOINT + utils.getOrigin()
-        }
-      ];
-    } else {
-      return [];
-    }
+  getUserSyncs: function() {
```
Comment (elberdev) [inline modules/yieldmoBidAdapter.js:108]: Do you know what the deal is with the space or no space before function parens inconsistency?

**Comment by MelodyLi2017** [comment]
updated not to have package lock changes

**Pull request merged**